src/Lykke.Job.BlockchainCashinDetector/Modules/JobModule.cs

```diff
@@ -22,6 +24,30 @@ public JobModule(IReloadingManager<AppSettings> settings)
 
         protected override void Load(ContainerBuilder builder)
         {
+            Lykke.Cqrs.MessageCancellation.Configuration.ContainerBuilderExtensions.RegisterCqrsMessageCancellation(
+            builder,
+            (options) =>
+            {
+                #region Registry
+
+                //Commands
+                options.Value
```

Comment:
Bad thing is - we can't be sure, that all messages are listed here. Will be good to implement some kind of verification